addons/aws/0.1.0/install.sh

```diff
@@ -9,7 +9,12 @@ function aws() {
         insert_resources "$dst/kustomization.yaml" storageclass.yaml
     fi
 
-    kubectl apply -k "$DIR/kustomize/aws/"
+    if  [ -s "$DIR/kustomize/aws/kustomization.yaml" ]; then
```

Comment:
basically checking if the file is empty , if the file is empty `kubectl apply -k` errors with `no objects`